src/main/java/org/folio/circulation/domain/CreateRequestService.java

```diff
@@ -155,4 +151,41 @@ private static RequestAndRelatedRecords setRequestQueuePosition(
   private static boolean isTheSameRequester(RequestAndRelatedRecords it, Request that) {
     return Objects.equals(it.getUserId(), that.getUserId());
   }
+  
+  private static CompletableFuture<Result<Loan>> findOpenLoanWithRequest(RequestAndRelatedRecords requestAndRelatedRecords,
+    LoanRepository loanRepository) {
+    Request request = requestAndRelatedRecords.getRequest();
+    return loanRepository.findOpenLoanForRequest(request);
+  }
+
+  private static Result<RequestAndRelatedRecords> refuseWhenLoanRequestFails(RequestAndRelatedRecords requestAndRelatedRecords,
+    Result<Loan> loanResult) {
+    if (loanResult.failed()) {
+      return Result.failed(loanResult.cause());
+    } else {
+      return Result.of(() -> requestAndRelatedRecords);
+    }
+  } 
+
+  private static Result<RequestAndRelatedRecords> refuseWhenUserHasAlreadyBeenLoanedItem(RequestAndRelatedRecords requestAndRelatedRecords,
+    Result<Loan> loanResult) {
+
+    Request request = requestAndRelatedRecords.getRequest();
+
+    Loan loan = loanResult.value();
+
+    String userId = request.getProxyUserId() == null ? request.getUserId() : request.getProxyUserId();
```

Comment:
The user making the request is always the `userId` and so this logic can be removed